script.js

```diff
@@ -1,53 +1,58 @@
-window.onload = () => {
-  let books = [];
+class Book {
+  static books = [];
+
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
 
-  const addBooksButton = document.querySelector('#add-book');
-  const bookTitle = document.querySelector('#book-title');
-  const bookAuthor = document.querySelector('#book-author');
+  addBook() {
+    Book.books.push(this);
+    Book.store(Book.books);
+  }
 
-  const store = (books) => {
+  static store(books) {
     localStorage.setItem('books', JSON.stringify(books));
-  };
-
-  const addBook = (title, author) => {
-    const newBook = {
-      title,
-      author,
-    };
-    books.push(newBook);
-    store(books);
-  };
-
-  const displayBooks = () => {
-    const booksLocalStorage = JSON.parse(localStorage.getItem('books'));
-    books = booksLocalStorage;
+  }
+
+  static displayBooks() {
+    Book.books = JSON.parse(localStorage.getItem('books'));
     let html = '';
-    booksLocalStorage.forEach((book, index) => {
-      html += `<div class="book">
+    if (Book.books) {
+      Book.books.forEach((book, index) => {
+        html += `<div class="book">
                 <h4>${book.title}</h4>
                 <p>${book.author}</p>
                 <button type="button" class="removeBtn" id="${index}">Remove</button>
                 <hr />
               </div>`;
-    });
-    document.querySelector('#books-container').innerHTML = html;
+      });
+      document.querySelector('#books-container').innerHTML = html;
 
-    const removeBook = document.querySelectorAll('.removeBtn');
+      const removeBook = document.querySelectorAll('.removeBtn');
 
-    [...removeBook].forEach((button) => {
-      button.addEventListener('click', (e) => {
-        books.splice(e.target.id, 1);
-        store(books);
-        displayBooks();
+      [...removeBook].forEach((button) => {
+        button.addEventListener('click', (e) => {
+          Book.books.splice(e.target.id, 1);
+          Book.store(Book.books);
+          Book.displayBooks();
+        });
       });
-    });
-  };
+    }
+  }
+}
+
+const addBooksButton = document.querySelector('#add-book');
+const bookTitle = document.querySelector('#book-title');
+const bookAuthor = document.querySelector('#book-author');
 
-  addBooksButton.addEventListener('click', (e) => {
-    e.preventDefault();
-    addBook(bookTitle.value, bookAuthor.value);
-    displayBooks();
-  });
+addBooksButton.addEventListener('click', (e) => {
+  e.preventDefault();
```

Comment:
- [ ] It would be better to target the `form` element itself and use the `submit` event for this action. This is part of best practices and makes your code cleaner.
- [ ] Also using the ` type='submit'` on the HTML form button would be a good fix

Reply:
Done